proc_scripts/load_data.py

```diff
@@ -214,13 +409,37 @@ def proc_DOSY_CPMG(s):
     # }}}
     return s
 
+def proc_ESR_linewidth(s):
+    logging.info("loading preprocessing for ESR linewidth calculation")
+    s -= s['$B_0$',:50].C.mean('$B_0$')
+    s_integral = s.C.run_nopop(np.cumsum,'$B_0$')
+    x1,x2 = s_integral.getaxis('$B_0$')[r_[5,-5]]
+    y1 = s_integral.data[:5].mean()
+    y2 = s_integral.data[-5:].mean()
+    straight_baseline = (s.fromaxis('$B_0$')-x1)*(y2-y1)/(x2-x1)
+    s_integral -= straight_baseline
+    s_integral /= s_integral.data.mean()
+    center_field = (s_integral * s.fromaxis('$B_0$')).mean('$B_0$').item()
+    s.setaxis('$B_0$',lambda x: x-center_field)
+    s_integral = s.C.run_nopop(np.cumsum,'$B_0$')
+    logger.info(strm(s_integral))
+    return s    
+
 postproc_dict = {'ag_IR2H':proc_bruker_deut_IR_withecho_mancyc,
         'ab_ir2h':proc_bruker_deut_IR_mancyc,
+        'ag_CPMG_strob':proc_bruker_CPMG_v1,
+        'ag_T1CPMG_2h':proc_bruker_T1CPMG_v1,
+        'chirp':proc_capture,
         'spincore_CPMG_v1':proc_spincore_CPMG_v1,
         'spincore_Hahn_echoph_v1':proc_Hahn_echoph,
-        'spincore_nutation_v1':proc_nutation,
         'spincore_IR_v1':proc_spincore_IR,
+        'spincore_nutation_v1':proc_nutation,
+        'spincore_nutation_v2':proc_nutation_amp,        
         'spincore_ODNP_v1':proc_spincore_ODNP_v1,
-        'square_wave_capture_v1':proc_square_wave_capture,
-        'DOSY_CPMG_v1':proc_DOSY_CPMG}
+        'spincore_var_tau_v1':proc_var_tau,
+        'square_wave_capture_v1':proc_capture,
+        'DOSY_CPMG_v1':proc_DOSY_CPMG,
+        'ESR_linewidth':proc_ESR_linewidth,
+
+}
```

Comment:
add a docstring at the beginning of the file that describes your convention (everything should be in the Δp domain rather than the φ domain -- is it in the t2 domain or the ν2 domain -- also, anyone who modifies this isn't allowed to do anything that's not invertible (anything that throws out information, like slicing, filtering, etc.))

Reply:
attempted